### PR TITLE
Ulrika and Alexandra

### PR DESCRIPTION
**Pull request by Cocofnas**

### CODE REVIEW

**Comment by Cocofnas** [comment]
https://stunning-faloodeh-3e176a.netlify.app/

File: src/components/QuestionAge.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        Please pick an age-span<br></br>
+      </label>
+      <select
+        id="ageSelect"
+        name="ageSelect"
+        value={selectedAge}
+        onChange={handleAgeChange}
+      >
+        <option value="">Age-span</option>
+        {ageSpans.map((ageSpan) => (
```
Comment (Hang-Nguyen-Vu) [inline src/components/QuestionAge.jsx:59]: This is a smart way to display the age spans without having to write them in the options one by one! Love it!

File: src/components/QuestionCelebrate.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  // Update the selectedOption state when the celebrateChristmas prop changes
+  useEffect(() => {
+    setSelectedOption(celebrateChristmas);
+  }, [celebrateChristmas]);
+
+  const handleRadioChange = (event) => {
+    const selectedOption = event.target.value;
+    setSelectedOption(selectedOption);
+    onAnswer(selectedOption);
```
Comment (Hang-Nguyen-Vu) [inline src/components/QuestionCelebrate.jsx:14]: I have never seen onAnswer! Great to learn sth new, thanks!

File: src/components/QuestionExcited.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div>
+      <h2>Are you excited about Christmas?</h2>
+      <p>How much on a scale from 1 to 100? Slide the bar to display your excitementlevel🎄⭐🎄 </p>
+      <input
+        type="range"
+        min="1"
+        max="100"
+        value={selectedLevel}
+        onChange={handleSliderChange}
```
Comment (Hang-Nguyen-Vu) [inline src/components/QuestionExcited.jsx:27]: Really cool with range slider!

**Review by Hang-Nguyen-Vu — COMMENTED** [review]
I really enjoyed filling out the survey 😊 The code is well structured and easy to follow. I love the way you organized data in objects and arrays. Well done!

**Pull request closed without merging**